conference/management/commands/create_initial_data_for_dev.py

```diff
@@ -52,11 +52,15 @@ def handle(self, *args, **options):
 
         print("Creating regular users")
         alice = AssopyUser.objects.create_user(
+            first_name="Alice",
+            last_name="Doe",
             email="alice@europython.eu",
             password="europython",
             active=True,
         )
         bob = AssopyUser.objects.create_user(
+            first_name="Bob",
```

Comment:
Thanks! Could you add first name and last name to the admin user and cesar user creation, too?

Reply:
Done. Adding first and last name to admin required changing the `create_superuser` function to accept them (https://github.com/EuroPython/epcon/blob/dev/ep2020/assopy/models.py#L184).

Reply:
Joe Schmoe :laughing: